tests/test_office_structure/test_delete_level.py

```diff
@@ -0,0 +1,23 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.office_structure.delete_level_fixtures import (
+  delete_level_fixtures, delete_level_response,
+  delete_level_fixtures_non_existant_id
+)
+
+
+class TestDeleteQuestion(BaseTestCase):
+    def test_delete_level(self):
+        """Test whether level has been deleted"""
```

Comment:
This should be `test_delete_level_works` and the docstring should be "Test that admin can delete a level"

Reply:
Thanks, @Bonifase, this has been implemented.